rinja_derive/src/lib.rs

```diff
@@ -311,9 +311,6 @@ const BUILT_IN_FILTERS: &[&str] = &[
     "into_f64",
     "into_isize",
     "join",
-    "linebreaks",
```

Comment:
Isn't it still a function in `generator.rs`?

Reply:
Oops, yes, of course.

Reply:
I sorted the list while re-adding the items. Looks nicer this way. :)